### PR TITLE
Route composer integration

### PR DESCRIPTION
**Pull request by Dm94st**
* `RouteComposer` has been integrated
* Horizontal dependencies has been removed(i.m `AddHost`, `AboutScreen`)
* Old router has been removed due to the new approach 
* `PostLaunch` module has been created
* Small bug fixes and improvements

### CODE REVIEW

File: Wake On Lan/Modules/PostLaunch/Sources/PostLaunchView.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    // MARK: - Appearance
+
+    private let appearance = Appearance(); struct Appearance {
+        /// Application main logo
+        let logoImage = Asset.Assets.owl.image
+    }
+
+    // MARK: - Properties
+
+    private lazy var logoImageView: UIImageView = {
```
Comment (Dm94st) [inline Wake On Lan/Modules/PostLaunch/Sources/PostLaunchView.swift:22]: It is awful solution as image does not change according to the current color scheme. It should be fixed in the next release/commit/PR.

File: Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenHeaderViewViewModel.swift [hunk 2]
```diff
@@ -5,7 +5,6 @@
 //  Created by Dmitry on 08.08.2021.
 //
 
-// sourcery: AutoEquatable
```
Comment (tr1ckyf0x) [inline Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenHeaderViewViewModel.swift:8]: It breaks tests

File: Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenViewController.swift [hunk 3]
```diff
@@ -6,9 +6,10 @@
 //  Copyright © 2021 Владислав Лисянский. All rights reserved.
 //
 
+import SharedRouter
 import UIKit
 
-public final class AboutScreenViewController: UIViewController {
+public final class AboutScreenViewController: UIViewController, Navigates {
```
Comment (tr1ckyf0x) [inline Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenViewController.swift:12]: What it is used for in ViewController?

File: Wake On Lan/Modules/AddHost/README.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
-# AddHost
+# PostLaunch
```
Comment (tr1ckyf0x) [inline Wake On Lan/Modules/AddHost/README.md:1]: Seems like a mistake

File: Wake On Lan/Modules/CoreDataService/Sources/Migrations/CoreDataAppToSharedGroupMigration.swift [hunk 5]
```diff
@@ -93,13 +93,14 @@ extension CoreDataAppToSharedGroupMigration {
                 try? fileManager.removeItem(at: url)
             }
         )
-        if let error {
+        if let error = error {
```
Comment (tr1ckyf0x) [inline Wake On Lan/Modules/CoreDataService/Sources/Migrations/CoreDataAppToSharedGroupMigration.swift:96]: Update your Xcode : )
```suggestion
        if let error {
```

File: Wake On Lan/Modules/CoreDataService/Sources/Models/Host.swift [hunk 6]
```diff
@@ -32,7 +32,7 @@ public final class Host: NSManagedObject, HostRepresentable {
 
     public private(set) var ipAddress: String? {
         get {
-            guard let ipAddressData else { return nil }
+            guard let ipAddressData = ipAddressData else { return nil }
```
Comment (tr1ckyf0x) [inline Wake On Lan/Modules/CoreDataService/Sources/Models/Host.swift:35]: The same. Update Xcode

File: Wake On Lan/Modules/PostLaunch/Sources/PostLaunchRoutes.swift [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+//  Created by Dmitry Stavitsky on 17.09.2022.
+//
+
+import SharedRouter
+
+public protocol PostLaunchRoutes {
+    /// Navigates to the list of hosts
+    func hostList() -> Route
+    /// Navigates to back
+    func back(animated: Bool) -> Route
```
Comment (tr1ckyf0x) [inline Wake On Lan/Modules/PostLaunch/Sources/PostLaunchRoutes.swift:14]: Why PostLaunch must have back?

File: Wake On Lan/Modules/WakeOnLanService/Sources/WakeOnLanService/MagicPacketBuilder.swift [hunk 8]
```diff
@@ -7,7 +7,7 @@ public final class MagicPacketBuilder {
 // MARK: - BuildsMagicPacket
 extension MagicPacketBuilder: BuildsMagicPacket {
     public func build(for macAddress: String?) throws -> [UInt8] {
-        guard let macAddress else { throw MagicPacketError.wrongMacAddressFormat }
+        guard let macAddress = macAddress else { throw MagicPacketError.wrongMacAddressFormat }
```
Comment (tr1ckyf0x) [inline Wake On Lan/Modules/WakeOnLanService/Sources/WakeOnLanService/MagicPacketBuilder.swift:10]: Update Xcode.

**Review by tr1ckyf0x — APPROVED** [review]

File: Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenViewController.swift (hunk 3 above)
Comment (Dm94st) [reply Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenViewController.swift:12]: No, it is was my mistake as it seems that I forgot to remove it.

File: Wake On Lan/Modules/AddHost/README.md (hunk 4 above)
Comment (Dm94st) [reply Wake On Lan/Modules/AddHost/README.md:1]: yeah

File: Wake On Lan/Modules/CoreDataService/Sources/Migrations/CoreDataAppToSharedGroupMigration.swift (hunk 5 above)
Comment (Dm94st) [reply Wake On Lan/Modules/CoreDataService/Sources/Migrations/CoreDataAppToSharedGroupMigration.swift:96]: I have already got the new Xcode version, actually I made this changes when only the Xcode 13 was installed on my mac.

File: Wake On Lan/Modules/PostLaunch/Sources/PostLaunchRoutes.swift (hunk 7 above)
Comment (Dm94st) [reply Wake On Lan/Modules/PostLaunch/Sources/PostLaunchRoutes.swift:14]: We can remove it, I think

**Pull request merged**